ui/src/translations/fr.json

```diff
@@ -135,5 +136,10 @@
   "please_refer_to": "Veuillez vous référer à",
   "import_solution_hint": "Vous devez importer une solution pour préparer votre environnement",
   "redirecting": "Redirection…",
-  "documentation": "Documentation"
+  "documentation": "Documentation",
+  "env_preparation": "Environment preparation",
```

Comment:
Probably not a correct French translation, what's the context/use for this string?
`Préparation de l’environnement` maybe